__tests__/app.test.js

```diff
@@ -183,3 +183,68 @@ test("404: Responds with error message if article does not exist", () => {
   })
 })
 })
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: Should add comment to an article when given article_id and the user is already defined in users.js", () => {
+    const comment = {
+      username: "rogersop",
+      body: "This is a bad article name"
+    }
+    return request(app)
+    .post("/api/articles/2/comments")
+    .expect(201)
+    .send(comment)
+    .then(({ body }) => {
+      const { comment } = body;
+      expect(comment).toMatchObject({
+        comment_id: expect.any(Number),
+        votes: expect.any(Number),
+        created_at: expect.any(String),
+        author: "rogersop",
+        body: "This is a bad article name",
+        article_id: 2
+      })
+    })
+  })
+  test("400: Sends appropriate status and error message when no username and/or body is given", () => {
+    const invalidComment = {
+      username: "user123"
+    }
+    return request(app)
+    .post("/api/articles/:article_id/comments")
```

Comment:
You should still use an actual article_id - that way we know the problem is the comment not the path